blazor/datagrid/adaptive-layout.md

```diff
@@ -72,7 +72,7 @@ The DataGrid will render the row elements vertically while setting the [RowRende
 @using Syncfusion.Blazor.Buttons
 
 <div style="position:relative; min-height: 500px;">
-    <SfGrid DataSource="@Orders" AllowSorting="true" AllowFiltering="true" EnableAdaptiveUI="true" Toolbar="@(new List<string>() { "Add", "Edit", "Delete", "Cancel", "Update", "Search" })" RowRenderingMode="RowDirection.Vertical" Height="100%" Width="100%" AllowPaging="true">
```

Comment:
https://blazor.syncfusion.com/documentation/datagrid/grouping#initial-group ---- use property binding similar to this.